2025-03-29 13:10:02.999666+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "\000a    create table books (\000a        id bigint generated by default as identity,\000a        author varchar(255),\000a        available boolean not null,\000a        title varchar(255),\000a        [*]year integer not null,\000a        primary key (id)\000a    )"; expected "identifier"; SQL statement:

    create table books (
        id bigint generated by default as identity,
        author varchar(255),
        available boolean not null,
        title varchar(255),
        year integer not null,
        primary key (id)
    ) [42001-224]
2025-03-29 13:10:03.006056+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "BOOKS" not found; SQL statement:

    alter table if exists loans 
       add constraint FKokwvlrv6o4i4h3le3bwhe6kie 
       foreign key (book_id) 
       references  [42102-224]
2025-03-29 13:12:20.491638+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "\000a    create table books (\000a        id bigint generated by default as identity,\000a        author varchar(255),\000a        available boolean not null,\000a        title varchar(255),\000a        [*]year integer not null,\000a        primary key (id)\000a    )"; expected "identifier"; SQL statement:

    create table books (
        id bigint generated by default as identity,
        author varchar(255),
        available boolean not null,
        title varchar(255),
        year integer not null,
        primary key (id)
    ) [42001-224]
2025-03-29 13:12:20.497706+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "BOOKS" not found; SQL statement:

    alter table if exists loans 
       add constraint FKokwvlrv6o4i4h3le3bwhe6kie 
       foreign key (book_id) 
       references  [42102-224]
2025-03-29 13:12:44.215958+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-03-29 13:16:00.377951+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "\000a    create table books (\000a        id bigint generated by default as identity,\000a        author varchar(255),\000a        available boolean not null,\000a        title varchar(255),\000a        [*]year integer not null,\000a        primary key (id)\000a    )"; expected "identifier"; SQL statement:

    create table books (
        id bigint generated by default as identity,
        author varchar(255),
        available boolean not null,
        title varchar(255),
        year integer not null,
        primary key (id)
    ) [42001-224]
2025-03-29 13:16:00.384162+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "BOOKS" not found; SQL statement:

    alter table if exists loans 
       add constraint FKokwvlrv6o4i4h3le3bwhe6kie 
       foreign key (book_id) 
       references  [42102-224]
2025-03-29 13:16:00.414224+01:00 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "\000a    create table books (\000a        id bigint generated by default as identity,\000a        author varchar(255),\000a        available boolean not null,\000a        title varchar(255),\000a        [*]year integer not null,\000a        primary key (id)\000a    )"; expected "identifier"; SQL statement:

    create table books (
        id bigint generated by default as identity,
        author varchar(255),
        available boolean not null,
        title varchar(255),
        year integer not null,
        primary key (id)
    ) [42001-224]
2025-03-29 13:16:00.415670+01:00 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "BOOKS" not found; SQL statement:

    alter table if exists loans 
       add constraint FKokwvlrv6o4i4h3le3bwhe6kie 
       foreign key (book_id) 
       references  [42102-224]
2025-03-29 13:16:00.442170+01:00 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "\000a    create table books (\000a        id bigint generated by default as identity,\000a        author varchar(255),\000a        available boolean not null,\000a        title varchar(255),\000a        [*]year integer not null,\000a        primary key (id)\000a    )"; expected "identifier"; SQL statement:

    create table books (
        id bigint generated by default as identity,
        author varchar(255),
        available boolean not null,
        title varchar(255),
        year integer not null,
        primary key (id)
    ) [42001-224]
2025-03-29 13:16:00.443197+01:00 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "BOOKS" not found; SQL statement:

    alter table if exists loans 
       add constraint FKokwvlrv6o4i4h3le3bwhe6kie 
       foreign key (book_id) 
       references  [42102-224]
2025-03-29 13:16:07.184339+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-03-29 13:16:07.188968+01:00 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "BOOKS" not found; SQL statement:
insert into books (author,available,title,year,id) values (?,?,?,?,default) [42102-224]
2025-03-29 13:16:08.543399+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-03-29 13:16:08.547002+01:00 jdbc[6]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "BOOKS" not found; SQL statement:
insert into books (author,available,title,year,id) values (?,?,?,?,default) [42102-224]
2025-03-29 13:16:08.959671+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-03-29 13:16:08.963169+01:00 jdbc[7]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "BOOKS" not found; SQL statement:
insert into books (author,available,title,year,id) values (?,?,?,?,default) [42102-224]
2025-03-29 13:16:09.994075+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-03-29 13:16:09.996743+01:00 jdbc[8]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "BOOKS" not found; SQL statement:
insert into books (author,available,title,year,id) values (?,?,?,?,default) [42102-224]
2025-03-29 13:16:10.175963+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-03-29 13:16:10.178739+01:00 jdbc[9]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "BOOKS" not found; SQL statement:
insert into books (author,available,title,year,id) values (?,?,?,?,default) [42102-224]
2025-03-29 13:16:10.323528+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-03-29 13:16:10.324408+01:00 jdbc[10]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "BOOKS" not found; SQL statement:
insert into books (author,available,title,year,id) values (?,?,?,?,default) [42102-224]
2025-03-29 13:16:10.509964+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-03-29 13:16:10.513108+01:00 jdbc[11]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "BOOKS" not found; SQL statement:
insert into books (author,available,title,year,id) values (?,?,?,?,default) [42102-224]
2025-03-29 13:16:11.576917+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-03-29 13:16:11.580457+01:00 jdbc[12]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "BOOKS" not found; SQL statement:
insert into books (author,available,title,year,id) values (?,?,?,?,default) [42102-224]
2025-03-29 13:16:11.777191+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-03-29 13:16:11.780173+01:00 jdbc[13]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "BOOKS" not found; SQL statement:
insert into books (author,available,title,year,id) values (?,?,?,?,default) [42102-224]
2025-03-29 13:16:11.957466+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2025-03-29 13:16:11.959134+01:00 jdbc[14]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "BOOKS" not found; SQL statement:
insert into books (author,available,title,year,id) values (?,?,?,?,default) [42102-224]
2025-03-29 13:25:54 jdbc[3]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1573)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:469)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:360)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:714)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:598)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:844)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:721)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:400)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:720)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
2025-03-29 13:25:54 jdbc[3]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1573)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:469)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:360)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:714)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:598)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:844)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:721)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:400)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:720)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
2025-03-29 13:36:32 jdbc[3]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1573)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:469)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:360)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:714)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:598)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:844)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:721)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:400)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:720)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
2025-03-29 13:37:12 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Tabulka "BOOK" nenalezena
Table "BOOK" not found; SQL statement:
SELECT *
FROM BOOK [42102-220]
2025-03-29 13:37:34 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Tabulka "BOOKS" nenalezena
Table "BOOKS" not found; SQL statement:
SELECT *
FROM BOOKS [42102-220]
2025-03-29 14:46:22.361973+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Chyba syntaxe v SQL příkazu "\000d\000a    create table books (\000d\000a        id bigint generated by default as identity,\000d\000a        author varchar(255),\000d\000a        available boolean not null,\000d\000a        title varchar(255),\000d\000a        [*]year integer not null,\000d\000a        primary key (id)\000d\000a    )"; očekáváno "identifier"
Syntax error in SQL statement "\000d\000a    create table books (\000d\000a        id bigint generated by default as identity,\000d\000a        author varchar(255),\000d\000a        available boolean not null,\000d\000a        title varchar(255),\000d\000a        [*]year integer not null,\000d\000a        primary key (id)\000d\000a    )"; expected "identifier"; SQL statement:

    create table books (
        id bigint generated by default as identity,
        author varchar(255),
        available boolean not null,
        title varchar(255),
        year integer not null,
        primary key (id)
    ) [42001-224]
2025-03-29 14:46:22.375977+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Tabulka "BOOKS" nenalezena
Table "BOOKS" not found; SQL statement:

    alter table if exists loans 
       add constraint FKokwvlrv6o4i4h3le3bwhe6kie 
       foreign key (book_id) 
       references [42102-224]
2025-03-29 14:46:22.436990+01:00 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Chyba syntaxe v SQL příkazu "\000d\000a    create table books (\000d\000a        id bigint generated by default as identity,\000d\000a        author varchar(255),\000d\000a        available boolean not null,\000d\000a        title varchar(255),\000d\000a        [*]year integer not null,\000d\000a        primary key (id)\000d\000a    )"; očekáváno "identifier"
Syntax error in SQL statement "\000d\000a    create table books (\000d\000a        id bigint generated by default as identity,\000d\000a        author varchar(255),\000d\000a        available boolean not null,\000d\000a        title varchar(255),\000d\000a        [*]year integer not null,\000d\000a        primary key (id)\000d\000a    )"; expected "identifier"; SQL statement:

    create table books (
        id bigint generated by default as identity,
        author varchar(255),
        available boolean not null,
        title varchar(255),
        year integer not null,
        primary key (id)
    ) [42001-224]
2025-03-29 14:46:22.440990+01:00 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Tabulka "BOOKS" nenalezena
Table "BOOKS" not found; SQL statement:

    alter table if exists loans 
       add constraint FKokwvlrv6o4i4h3le3bwhe6kie 
       foreign key (book_id) 
       references [42102-224]
2025-03-29 14:46:22.571020+01:00 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Chyba syntaxe v SQL příkazu "\000d\000a    create table books (\000d\000a        id bigint generated by default as identity,\000d\000a        author varchar(255),\000d\000a        available boolean not null,\000d\000a        title varchar(255),\000d\000a        [*]year integer not null,\000d\000a        primary key (id)\000d\000a    )"; očekáváno "identifier"
Syntax error in SQL statement "\000d\000a    create table books (\000d\000a        id bigint generated by default as identity,\000d\000a        author varchar(255),\000d\000a        available boolean not null,\000d\000a        title varchar(255),\000d\000a        [*]year integer not null,\000d\000a        primary key (id)\000d\000a    )"; expected "identifier"; SQL statement:

    create table books (
        id bigint generated by default as identity,
        author varchar(255),
        available boolean not null,
        title varchar(255),
        year integer not null,
        primary key (id)
    ) [42001-224]
2025-03-29 14:46:22.574020+01:00 jdbc[5]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Tabulka "BOOKS" nenalezena
Table "BOOKS" not found; SQL statement:

    alter table if exists loans 
       add constraint FKokwvlrv6o4i4h3le3bwhe6kie 
       foreign key (book_id) 
       references [42102-224]
2025-03-29 14:46:25.523791+01:00 jdbc[6]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Chyba syntaxe v SQL příkazu "\000d\000a    create table books (\000d\000a        id bigint generated by default as identity,\000d\000a        author varchar(255),\000d\000a        available boolean not null,\000d\000a        title varchar(255),\000d\000a        [*]year integer not null,\000d\000a        primary key (id)\000d\000a    )"; očekáváno "identifier"
Syntax error in SQL statement "\000d\000a    create table books (\000d\000a        id bigint generated by default as identity,\000d\000a        author varchar(255),\000d\000a        available boolean not null,\000d\000a        title varchar(255),\000d\000a        [*]year integer not null,\000d\000a        primary key (id)\000d\000a    )"; expected "identifier"; SQL statement:

    create table books (
        id bigint generated by default as identity,
        author varchar(255),
        available boolean not null,
        title varchar(255),
        year integer not null,
        primary key (id)
    ) [42001-224]
2025-03-29 14:46:25.525805+01:00 jdbc[6]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Tabulka "BOOKS" nenalezena
Table "BOOKS" not found; SQL statement:

    alter table if exists loans 
       add constraint FKokwvlrv6o4i4h3le3bwhe6kie 
       foreign key (book_id) 
       references [42102-224]
2025-03-29 14:46:25.583712+01:00 jdbc[7]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Chyba syntaxe v SQL příkazu "\000d\000a    create table books (\000d\000a        id bigint generated by default as identity,\000d\000a        author varchar(255),\000d\000a        available boolean not null,\000d\000a        title varchar(255),\000d\000a        [*]year integer not null,\000d\000a        primary key (id)\000d\000a    )"; očekáváno "identifier"
Syntax error in SQL statement "\000d\000a    create table books (\000d\000a        id bigint generated by default as identity,\000d\000a        author varchar(255),\000d\000a        available boolean not null,\000d\000a        title varchar(255),\000d\000a        [*]year integer not null,\000d\000a        primary key (id)\000d\000a    )"; expected "identifier"; SQL statement:

    create table books (
        id bigint generated by default as identity,
        author varchar(255),
        available boolean not null,
        title varchar(255),
        year integer not null,
        primary key (id)
    ) [42001-224]
2025-03-29 14:46:25.586912+01:00 jdbc[7]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Tabulka "BOOKS" nenalezena
Table "BOOKS" not found; SQL statement:

    alter table if exists loans 
       add constraint FKokwvlrv6o4i4h3le3bwhe6kie 
       foreign key (book_id) 
       references [42102-224]
2025-03-29 14:46:25.642902+01:00 jdbc[8]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Chyba syntaxe v SQL příkazu "\000d\000a    create table books (\000d\000a        id bigint generated by default as identity,\000d\000a        author varchar(255),\000d\000a        available boolean not null,\000d\000a        title varchar(255),\000d\000a        [*]year integer not null,\000d\000a        primary key (id)\000d\000a    )"; očekáváno "identifier"
Syntax error in SQL statement "\000d\000a    create table books (\000d\000a        id bigint generated by default as identity,\000d\000a        author varchar(255),\000d\000a        available boolean not null,\000d\000a        title varchar(255),\000d\000a        [*]year integer not null,\000d\000a        primary key (id)\000d\000a    )"; expected "identifier"; SQL statement:

    create table books (
        id bigint generated by default as identity,
        author varchar(255),
        available boolean not null,
        title varchar(255),
        year integer not null,
        primary key (id)
    ) [42001-224]
2025-03-29 14:46:25.645903+01:00 jdbc[8]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Tabulka "BOOKS" nenalezena
Table "BOOKS" not found; SQL statement:

    alter table if exists loans 
       add constraint FKokwvlrv6o4i4h3le3bwhe6kie 
       foreign key (book_id) 
       references [42102-224]
2025-03-29 13:50:33 database: flush
org.h2.message.DbException: Obecná chyba: "org.h2.mvstore.MVStoreException: The file is locked: C:/Users/fstup/IdeaProjects/LibraryDb/data/librarydb.mv.db [2.2.220/7]"
General error: "org.h2.mvstore.MVStoreException: The file is locked: C:/Users/fstup/IdeaProjects/LibraryDb/data/librarydb.mv.db [2.2.220/7]" [50000-220]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:407)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:122)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:1547)
	at org.h2.mvstore.MVStore.panic(MVStore.java:371)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:291)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2036)
	at org.h2.mvstore.db.Store.<init>(Store.java:133)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at com.intellij.database.remote.jdbc.helpers.JdbcHelperImpl.connect(JdbcHelperImpl.java:790)
	at com.intellij.database.remote.jdbc.impl.RemoteDriverImpl.connect(RemoteDriverImpl.java:153)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:360)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:714)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:598)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:844)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:721)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:400)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:720)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: Obecná chyba: "org.h2.mvstore.MVStoreException: The file is locked: C:/Users/fstup/IdeaProjects/LibraryDb/data/librarydb.mv.db [2.2.220/7]"
General error: "org.h2.mvstore.MVStoreException: The file is locked: C:/Users/fstup/IdeaProjects/LibraryDb/data/librarydb.mv.db [2.2.220/7]" [50000-220]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 32 more
Caused by: org.h2.mvstore.MVStoreException: The file is locked: C:/Users/fstup/IdeaProjects/LibraryDb/data/librarydb.mv.db [2.2.220/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.SingleFileStore.lockFileChannel(SingleFileStore.java:143)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:117)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:81)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:286)
	... 26 more
2025-03-29 13:50:47 jdbc[3]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1573)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:469)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:360)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:714)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:598)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:844)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:721)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:400)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:720)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
2025-03-29 13:53:21 jdbc[3]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1573)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:469)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:360)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:714)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:598)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:844)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:721)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:400)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:720)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
2025-03-29 14:56:16.261364+01:00 database: flush
org.h2.message.DbException: Obecná chyba: "org.h2.mvstore.MVStoreException: The file is locked: C:/Users/fstup/IdeaProjects/LibraryDb/data/librarydb.mv.db [2.2.224/7]"
General error: "org.h2.mvstore.MVStoreException: The file is locked: C:/Users/fstup/IdeaProjects/LibraryDb/data/librarydb.mv.db [2.2.224/7]" [50000-224]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:407)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:122)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:1546)
	at org.h2.mvstore.MVStore.panic(MVStore.java:371)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:291)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2035)
	at org.h2.mvstore.db.Store.<init>(Store.java:133)
	at org.h2.engine.Database.<init>(Database.java:326)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:343)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:125)
	at org.h2.Driver.connect(Driver.java:59)
	at org.hibernate.orm.core@6.4.2.Final/org.hibernate.engine.jdbc.connections.internal.DriverConnectionCreator.makeConnection(DriverConnectionCreator.java:40)
	at org.hibernate.orm.core@6.4.2.Final/org.hibernate.engine.jdbc.connections.internal.BasicConnectionCreator.createConnection(BasicConnectionCreator.java:61)
	at org.hibernate.orm.core@6.4.2.Final/org.hibernate.engine.jdbc.connections.internal.DriverManagerConnectionProviderImpl$PooledConnections.addConnections(DriverManagerConnectionProviderImpl.java:499)
	at org.hibernate.orm.core@6.4.2.Final/org.hibernate.engine.jdbc.connections.internal.DriverManagerConnectionProviderImpl$PooledConnections.<init>(DriverManagerConnectionProviderImpl.java:372)
	at org.hibernate.orm.core@6.4.2.Final/org.hibernate.engine.jdbc.connections.internal.DriverManagerConnectionProviderImpl$PooledConnections$Builder.build(DriverManagerConnectionProviderImpl.java:550)
	at org.hibernate.orm.core@6.4.2.Final/org.hibernate.engine.jdbc.connections.internal.DriverManagerConnectionProviderImpl.buildPool(DriverManagerConnectionProviderImpl.java:102)
	at org.hibernate.orm.core@6.4.2.Final/org.hibernate.engine.jdbc.connections.internal.DriverManagerConnectionProviderImpl.configure(DriverManagerConnectionProviderImpl.java:82)
	at org.hibernate.orm.core@6.4.2.Final/org.hibernate.boot.registry.internal.StandardServiceRegistryImpl.configureService(StandardServiceRegistryImpl.java:136)
	at org.hibernate.orm.core@6.4.2.Final/org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:247)
	at org.hibernate.orm.core@6.4.2.Final/org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:215)
	at org.hibernate.orm.core@6.4.2.Final/org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.buildJdbcConnectionAccess(JdbcEnvironmentInitiator.java:395)
	at org.hibernate.orm.core@6.4.2.Final/org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.getJdbcEnvironmentUsingJdbcMetadata(JdbcEnvironmentInitiator.java:262)
	at org.hibernate.orm.core@6.4.2.Final/org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.initiateService(JdbcEnvironmentInitiator.java:107)
	at org.hibernate.orm.core@6.4.2.Final/org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.initiateService(JdbcEnvironmentInitiator.java:68)
	at org.hibernate.orm.core@6.4.2.Final/org.hibernate.boot.registry.internal.StandardServiceRegistryImpl.initiateService(StandardServiceRegistryImpl.java:130)
	at org.hibernate.orm.core@6.4.2.Final/org.hibernate.service.internal.AbstractServiceRegistryImpl.createService(AbstractServiceRegistryImpl.java:263)
	at org.hibernate.orm.core@6.4.2.Final/org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:238)
	at org.hibernate.orm.core@6.4.2.Final/org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:215)
	at org.hibernate.orm.core@6.4.2.Final/org.hibernate.boot.model.relational.Database.<init>(Database.java:45)
	at org.hibernate.orm.core@6.4.2.Final/org.hibernate.boot.internal.InFlightMetadataCollectorImpl.getDatabase(InFlightMetadataCollectorImpl.java:223)
	at org.hibernate.orm.core@6.4.2.Final/org.hibernate.boot.internal.InFlightMetadataCollectorImpl.<init>(InFlightMetadataCollectorImpl.java:191)
	at org.hibernate.orm.core@6.4.2.Final/org.hibernate.boot.model.process.spi.MetadataBuildingProcess.complete(MetadataBuildingProcess.java:170)
	at org.hibernate.orm.core@6.4.2.Final/org.hibernate.jpa.boot.internal.EntityManagerFactoryBuilderImpl.metadata(EntityManagerFactoryBuilderImpl.java:1432)
	at org.hibernate.orm.core@6.4.2.Final/org.hibernate.jpa.boot.internal.EntityManagerFactoryBuilderImpl.build(EntityManagerFactoryBuilderImpl.java:1503)
	at org.hibernate.orm.core@6.4.2.Final/org.hibernate.jpa.HibernatePersistenceProvider.createEntityManagerFactory(HibernatePersistenceProvider.java:55)
	at jakarta.persistence@3.1.0/jakarta.persistence.Persistence.createEntityManagerFactory(Persistence.java:80)
	at jakarta.persistence@3.1.0/jakarta.persistence.Persistence.createEntityManagerFactory(Persistence.java:55)
	at org.example.librarydb/org.example.librarydb.service.ReaderService.<init>(ReaderService.java:14)
	at org.example.librarydb/org.example.librarydb.HelloController.<init>(HelloController.java:22)
	at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:62)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:502)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:486)
	at javafx.fxml@21/javafx.fxml.FXMLLoader$ValueElement.processAttribute(FXMLLoader.java:941)
	at javafx.fxml@21/javafx.fxml.FXMLLoader$InstanceDeclarationElement.processAttribute(FXMLLoader.java:983)
	at javafx.fxml@21/javafx.fxml.FXMLLoader$Element.processStartElement(FXMLLoader.java:230)
	at javafx.fxml@21/javafx.fxml.FXMLLoader$ValueElement.processStartElement(FXMLLoader.java:757)
	at javafx.fxml@21/javafx.fxml.FXMLLoader.processStartElement(FXMLLoader.java:2853)
	at javafx.fxml@21/javafx.fxml.FXMLLoader.loadImpl(FXMLLoader.java:2649)
	at javafx.fxml@21/javafx.fxml.FXMLLoader.loadImpl(FXMLLoader.java:2563)
	at javafx.fxml@21/javafx.fxml.FXMLLoader.load(FXMLLoader.java:2531)
	at org.example.librarydb/org.example.librarydb.LibraryApp.start(LibraryApp.java:13)
	at javafx.graphics@21/com.sun.javafx.application.LauncherImpl.lambda$launchApplication1$9(LauncherImpl.java:839)
	at javafx.graphics@21/com.sun.javafx.application.PlatformImpl.lambda$runAndWait$12(PlatformImpl.java:483)
	at javafx.graphics@21/com.sun.javafx.application.PlatformImpl.lambda$runLater$10(PlatformImpl.java:456)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:400)
	at javafx.graphics@21/com.sun.javafx.application.PlatformImpl.lambda$runLater$11(PlatformImpl.java:455)
	at javafx.graphics@21/com.sun.glass.ui.InvokeLaterDispatcher$Future.run(InvokeLaterDispatcher.java:95)
	at javafx.graphics@21/com.sun.glass.ui.win.WinApplication._runLoop(Native Method)
	at javafx.graphics@21/com.sun.glass.ui.win.WinApplication.lambda$runLoop$3(WinApplication.java:185)
	at java.base/java.lang.Thread.run(Thread.java:1570)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: Obecná chyba: "org.h2.mvstore.MVStoreException: The file is locked: C:/Users/fstup/IdeaProjects/LibraryDb/data/librarydb.mv.db [2.2.224/7]"
General error: "org.h2.mvstore.MVStoreException: The file is locked: C:/Users/fstup/IdeaProjects/LibraryDb/data/librarydb.mv.db [2.2.224/7]" [50000-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 65 more
Caused by: org.h2.mvstore.MVStoreException: The file is locked: C:/Users/fstup/IdeaProjects/LibraryDb/data/librarydb.mv.db [2.2.224/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.SingleFileStore.lockFileChannel(SingleFileStore.java:143)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:117)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:81)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:286)
	... 59 more
2025-03-29 14:56:16.346038+01:00 database: flush
org.h2.message.DbException: Obecná chyba: "org.h2.mvstore.MVStoreException: The file is locked: C:/Users/fstup/IdeaProjects/LibraryDb/data/librarydb.mv.db [2.2.224/7]"
General error: "org.h2.mvstore.MVStoreException: The file is locked: C:/Users/fstup/IdeaProjects/LibraryDb/data/librarydb.mv.db [2.2.224/7]" [50000-224]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:407)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:122)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:1546)
	at org.h2.mvstore.MVStore.panic(MVStore.java:371)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:291)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2035)
	at org.h2.mvstore.db.Store.<init>(Store.java:133)
	at org.h2.engine.Database.<init>(Database.java:326)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:343)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:125)
	at org.h2.Driver.connect(Driver.java:59)
	at org.hibernate.orm.core@6.4.2.Final/org.hibernate.engine.jdbc.connections.internal.DriverConnectionCreator.makeConnection(DriverConnectionCreator.java:40)
	at org.hibernate.orm.core@6.4.2.Final/org.hibernate.engine.jdbc.connections.internal.BasicConnectionCreator.createConnection(BasicConnectionCreator.java:61)
	at org.hibernate.orm.core@6.4.2.Final/org.hibernate.engine.jdbc.connections.internal.DriverManagerConnectionProviderImpl$PooledConnections.addConnections(DriverManagerConnectionProviderImpl.java:499)
	at org.hibernate.orm.core@6.4.2.Final/org.hibernate.engine.jdbc.connections.internal.DriverManagerConnectionProviderImpl$PooledConnections.<init>(DriverManagerConnectionProviderImpl.java:372)
	at org.hibernate.orm.core@6.4.2.Final/org.hibernate.engine.jdbc.connections.internal.DriverManagerConnectionProviderImpl$PooledConnections$Builder.build(DriverManagerConnectionProviderImpl.java:550)
	at org.hibernate.orm.core@6.4.2.Final/org.hibernate.engine.jdbc.connections.internal.DriverManagerConnectionProviderImpl.buildPool(DriverManagerConnectionProviderImpl.java:102)
	at org.hibernate.orm.core@6.4.2.Final/org.hibernate.engine.jdbc.connections.internal.DriverManagerConnectionProviderImpl.configure(DriverManagerConnectionProviderImpl.java:82)
	at org.hibernate.orm.core@6.4.2.Final/org.hibernate.boot.registry.internal.StandardServiceRegistryImpl.configureService(StandardServiceRegistryImpl.java:136)
	at org.hibernate.orm.core@6.4.2.Final/org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:247)
	at org.hibernate.orm.core@6.4.2.Final/org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:215)
	at org.hibernate.orm.core@6.4.2.Final/org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.buildJdbcConnectionAccess(JdbcEnvironmentInitiator.java:395)
	at org.hibernate.orm.core@6.4.2.Final/org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.getJdbcEnvironmentUsingJdbcMetadata(JdbcEnvironmentInitiator.java:262)
	at org.hibernate.orm.core@6.4.2.Final/org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.initiateService(JdbcEnvironmentInitiator.java:107)
	at org.hibernate.orm.core@6.4.2.Final/org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator.initiateService(JdbcEnvironmentInitiator.java:68)
	at org.hibernate.orm.core@6.4.2.Final/org.hibernate.boot.registry.internal.StandardServiceRegistryImpl.initiateService(StandardServiceRegistryImpl.java:130)
	at org.hibernate.orm.core@6.4.2.Final/org.hibernate.service.internal.AbstractServiceRegistryImpl.createService(AbstractServiceRegistryImpl.java:263)
	at org.hibernate.orm.core@6.4.2.Final/org.hibernate.service.internal.AbstractServiceRegistryImpl.initializeService(AbstractServiceRegistryImpl.java:238)
	at org.hibernate.orm.core@6.4.2.Final/org.hibernate.service.internal.AbstractServiceRegistryImpl.getService(AbstractServiceRegistryImpl.java:215)
	at org.hibernate.orm.core@6.4.2.Final/org.hibernate.boot.model.relational.Database.<init>(Database.java:45)
	at org.hibernate.orm.core@6.4.2.Final/org.hibernate.boot.internal.InFlightMetadataCollectorImpl.getDatabase(InFlightMetadataCollectorImpl.java:223)
	at org.hibernate.orm.core@6.4.2.Final/org.hibernate.boot.internal.InFlightMetadataCollectorImpl.<init>(InFlightMetadataCollectorImpl.java:191)
	at org.hibernate.orm.core@6.4.2.Final/org.hibernate.boot.model.process.spi.MetadataBuildingProcess.complete(MetadataBuildingProcess.java:170)
	at org.hibernate.orm.core@6.4.2.Final/org.hibernate.jpa.boot.internal.EntityManagerFactoryBuilderImpl.metadata(EntityManagerFactoryBuilderImpl.java:1432)
	at org.hibernate.orm.core@6.4.2.Final/org.hibernate.jpa.boot.internal.EntityManagerFactoryBuilderImpl.build(EntityManagerFactoryBuilderImpl.java:1503)
	at org.hibernate.orm.core@6.4.2.Final/org.hibernate.jpa.HibernatePersistenceProvider.createEntityManagerFactory(HibernatePersistenceProvider.java:55)
	at jakarta.persistence@3.1.0/jakarta.persistence.Persistence.createEntityManagerFactory(Persistence.java:80)
	at jakarta.persistence@3.1.0/jakarta.persistence.Persistence.createEntityManagerFactory(Persistence.java:55)
	at org.example.librarydb/org.example.librarydb.service.ReaderService.<init>(ReaderService.java:14)
	at org.example.librarydb/org.example.librarydb.HelloController.<init>(HelloController.java:22)
	at org.example.librarydb/org.example.librarydb.LibraryApp.stop(LibraryApp.java:21)
	at javafx.graphics@21/com.sun.javafx.application.LauncherImpl.lambda$launchApplication1$10(LauncherImpl.java:858)
	at javafx.graphics@21/com.sun.javafx.application.PlatformImpl.lambda$runAndWait$12(PlatformImpl.java:483)
	at javafx.graphics@21/com.sun.javafx.application.PlatformImpl.lambda$runLater$10(PlatformImpl.java:456)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:400)
	at javafx.graphics@21/com.sun.javafx.application.PlatformImpl.lambda$runLater$11(PlatformImpl.java:455)
	at javafx.graphics@21/com.sun.glass.ui.InvokeLaterDispatcher$Future.run(InvokeLaterDispatcher.java:95)
	at javafx.graphics@21/com.sun.glass.ui.win.WinApplication._runLoop(Native Method)
	at javafx.graphics@21/com.sun.glass.ui.win.WinApplication.lambda$runLoop$3(WinApplication.java:185)
	at java.base/java.lang.Thread.run(Thread.java:1570)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: Obecná chyba: "org.h2.mvstore.MVStoreException: The file is locked: C:/Users/fstup/IdeaProjects/LibraryDb/data/librarydb.mv.db [2.2.224/7]"
General error: "org.h2.mvstore.MVStoreException: The file is locked: C:/Users/fstup/IdeaProjects/LibraryDb/data/librarydb.mv.db [2.2.224/7]" [50000-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 54 more
Caused by: org.h2.mvstore.MVStoreException: The file is locked: C:/Users/fstup/IdeaProjects/LibraryDb/data/librarydb.mv.db [2.2.224/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.SingleFileStore.lockFileChannel(SingleFileStore.java:143)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:117)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:81)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:286)
	... 48 more
2025-03-29 14:57:06.193017+01:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Tento objekt byl již uzavřen
The object is already closed [90007-224]
2025-03-29 14:57:06.206021+01:00 jdbc[4]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Tento objekt byl již uzavřen
The object is already closed [90007-224]
2025-03-29 14:57:06.448120+01:00 jdbc[5]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Tento objekt byl již uzavřen
The object is already closed [90007-224]
2025-03-29 13:58:37 database: flush
org.h2.message.DbException: Obecná chyba: "org.h2.mvstore.MVStoreException: The file is locked: C:/Users/fstup/IdeaProjects/LibraryDb/data/librarydb.mv.db [2.2.220/7]"
General error: "org.h2.mvstore.MVStoreException: The file is locked: C:/Users/fstup/IdeaProjects/LibraryDb/data/librarydb.mv.db [2.2.220/7]" [50000-220]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:407)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:122)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:1547)
	at org.h2.mvstore.MVStore.panic(MVStore.java:371)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:291)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2036)
	at org.h2.mvstore.db.Store.<init>(Store.java:133)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at com.intellij.database.remote.jdbc.helpers.JdbcHelperImpl.connect(JdbcHelperImpl.java:790)
	at com.intellij.database.remote.jdbc.impl.RemoteDriverImpl.connect(RemoteDriverImpl.java:153)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:360)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:714)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:598)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:844)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:721)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:400)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:720)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: Obecná chyba: "org.h2.mvstore.MVStoreException: The file is locked: C:/Users/fstup/IdeaProjects/LibraryDb/data/librarydb.mv.db [2.2.220/7]"
General error: "org.h2.mvstore.MVStoreException: The file is locked: C:/Users/fstup/IdeaProjects/LibraryDb/data/librarydb.mv.db [2.2.220/7]" [50000-220]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 32 more
Caused by: org.h2.mvstore.MVStoreException: The file is locked: C:/Users/fstup/IdeaProjects/LibraryDb/data/librarydb.mv.db [2.2.220/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.SingleFileStore.lockFileChannel(SingleFileStore.java:143)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:117)
	at org.h2.mvstore.SingleFileStore.open(SingleFileStore.java:81)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:286)
	... 26 more
2025-03-29 13:58:47 jdbc[3]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1573)
	at com.intellij.database.remote.jdbc.impl.RemoteConnectionImpl.setClientInfo(RemoteConnectionImpl.java:469)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at java.rmi/sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:360)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:200)
	at java.rmi/sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:714)
	at java.rmi/sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:598)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:844)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:721)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:400)
	at java.rmi/sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:720)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
